composer/datasets/hparams.py

```diff
@@ -73,7 +73,7 @@ class DatasetHparams(hp.Hparams, abc.ABC, metaclass=metaclass):
     def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHparams) -> Union[DataLoader, DataSpec]:
         """Creates a :class:`DataLoader` or :class:`DataloaderSpec` for this dataset.
 
-        Parameters:
+        Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.